src/View/SSViewer_DataPresenter.php

```diff
@@ -172,6 +172,10 @@ public function getInjectedValue($property, array $params, $cast = true)
         // Get source for this value
         $result = $this->getValueSource($property);
         if (!array_key_exists('source', $result)) {
+            // $Me is a special property. If nothing is providing an override, return the current item.
+            if ($property === 'Me') {
+                return ['obj' => $this->getItem()];
+            }
```

Comment:
I originally was going to have this return early - but I think it's worth allowing people to implement their own `Me` method or template provider in case someone's doing something really funky and custom.

Ultimately the reason we're moving this here at all is to handle cases where the current item isn't an instance of `ViewableData`, so anything people are doing on their own classes won't be affected either way.